enterprise/coderd/groups.go

```diff
@@ -171,6 +172,11 @@ func (api *API) patchGroup(rw http.ResponseWriter, r *http.Request) {
 			})
 			return
 		}
+		// Skip membership checks for the prebuilds user.
+		if id == prebuilds.SystemUserID.String() {
+			api.Logger.Debug(ctx, "Skipping membership check for prebuilds user")
```

Comment:
Can probably drop this log TBH; it won't have much operational value to operators.
```suggestion
```